server/http.go

```diff
@@ -53,13 +67,142 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 		p.completeUserOAuthToZoom(w, r)
 	case "/deauthorization":
 		p.deauthorizeUser(w, r)
+	case "/api/v1/updatePMI":
+		p.setPMI(w, r)
+	case "/api/v1/askPMI":
+		p.askPMI(w, r)
 	default:
 		http.NotFound(w, r)
 	}
 }
 
+func (p *Plugin) askPMI(w http.ResponseWriter, r *http.Request) {
+	response := &model.PostActionIntegrationResponse{}
+	decoder := json.NewDecoder(r.Body)
+	postActionIntegrationRequest := &model.PostActionIntegrationRequest{}
+	if err := decoder.Decode(&postActionIntegrationRequest); err != nil {
+		p.API.LogError("Error decoding PostActionIntegrationRequest params.", "Error", err.Error())
+		w.Header().Set("Content-Type", "application/json")
+		if err := json.NewEncoder(w).Encode(response); err != nil {
+			http.Error(w, "failed to write the response", http.StatusInternalServerError)
+		}
+		return
+	}
+
+	action := postActionIntegrationRequest.Context[Action].(string)
+	userID := postActionIntegrationRequest.Context[UserID].(string)
+	channelID := postActionIntegrationRequest.Context[ChannelID].(string)
+
+	slackAttachment := model.SlackAttachment{
+		Text: fmt.Sprintf("You have selected `%s` to start the meeting.", action),
+	}
+
+	post := &model.Post{
+		ChannelId: channelID,
+		UserId:    p.botUserID,
+		Id:        postActionIntegrationRequest.PostId,
+		CreateAt:  time.Now().Unix() * 1000,
+	}
+
+	model.ParseSlackAttachment(post, []*model.SlackAttachment{&slackAttachment})
+	p.API.UpdateEphemeralPost(userID, post)
+
+	w.Header().Set("Content-Type", "application/json")
+	if err := json.NewEncoder(w).Encode(post); err != nil {
+		http.Error(w, "failed to write the response", http.StatusInternalServerError)
+	}
+
+	p.startMeeting(action, userID, channelID)
+}
+
+func (p *Plugin) startMeeting(action string, userID string, channelID string) {
+	user, appErr := p.API.GetUser(userID)
+	if appErr != nil {
+		p.API.LogWarn("failed to get the user from userID", "error", appErr.Error())
+		return
+	}
+	zoomUser, authErr := p.authenticateAndFetchZoomUser(user)
+	if authErr != nil {
+		p.API.LogWarn("failed to authenticate and fetch the zoom user", "error", appErr.Error())
```

Comment:
```suggestion
		p.API.LogWarn("failed to authenticate and fetch the zoom user", "Error", appErr.Error())
```
Change this everywhere.